lib/rialto/etl/extractors/web_of_science.rb

```diff
@@ -9,15 +9,16 @@ module Extractors
       class WebOfScience
         DEFAULT_INSTITUTION = 'Stanford University'
 
-        attr_reader :client
+        attr_reader :client, :institution, :since
 
-        # @param [Hash] options
-        # @option options [String] :client a preconfigured client.  May be used for testing, all other
-        #                                  options will be ignored.
-        # @option options [String] :institution ('Stanford University') The institution to search for
-        # @option options [String] :since (nil) How far back to retrieve records. If not provided, extract all records.
-        def initialize(**options)
-          @client = options.fetch(:client) { build_client(options) }
+        # @param client [String] a preconfigured client. May be used for testing, all other options will be ignored.
+        # @param institution [String] The institution to search for (default: 'Stanford University')
+        # @param since [String] How far back to retrieve records. If not provided, extract all records. (default: nil)
+        def initialize(client: nil, institution: DEFAULT_INSTITUTION, since: nil)
+          @institution = institution
```

Comment:
I think you want `institution || DEFAULT_INSTITUTION` here, because if you call `WebOfScience.new(institution:nil)`, then `institution` will be `nil` here and that wasn't how the code worked before.

Reply:
Oops, sorry, I was confused between `WebOfScienceClient` and `WebOfScience`. Nevermind.